dbt/profiles.yml

```diff
@@ -22,7 +22,7 @@ athena:
       region_name: us-east-1
       schema: z_static_unused_dbt_stub_database
       database: awsdatacatalog
-      spark_work_group: primary-spark
+      spark_work_group: primary-spark-staging
```

Comment:
See [here](https://github.com/ccao-data/data-architecture/actions/runs/9306870644/job/25616957485#step:8:55) for a workflow run demonstrating that Python models still work using this workgroup.